### ISSUE TITLE
Accordion box heading loses focus after opening and closing

### ISSUE THREAD

**Issue by terracoda** [phetsims/plinko-probability#107]
I just noticed that after opening or closing the accordion box (Lab Screen), keyboard focus is lost.  Keyboard should remain on the heading of the box when the box is opened and closed. This allows the interaction tp be repeated, if necessary, and  also ensures visual focus in the sim is not lost.

**Question:**
This sim is just undergoing early keyboard navigation work, so maybe this is just not yet implemented?

@jessegreenberg, asssigning to you, and please re-assign to appropriate team member.

**Comment by pixelzoom** [comment]
@jessegreenberg This seems like an issue that should be in sun, where AccordionBox lives.  Or is there something sim-specific about this issue?

**Comment by terracoda** [comment]
@pixelzoom, great question, Chris. This is an old issue, and I just re-tested quickly in 3 browsers and the focus highlight is working fine.

I'll close this issue and if it come up again, we can post in this appropriate repo.
